.gitattributes

```diff
@@ -1,6 +1,9 @@
 /.editorconfig  export-ignore
-/.gitignore     export-ignore
 /.gitattributes export-ignore
+/.gitignore     export-ignore
+/.travis.yml    export-ignore
 /phpcs.php_cs   export-ignore
 /phpstan.neon   export-ignore
+/phpunit.xml    export-ignore
+/scripts        export-ignore
```

Comment:
The scripts can be used to access the transliteration features on the command line. The others make sense to exclude here, but I think that one ought to be included.

Reply:
Done.

Reply:
Awesome, thanks!